src/plothist/histogramming.py

```diff
@@ -13,7 +21,7 @@ def create_axis(bins, range=None, data=np.array([])):
     bins : int or array-like
         The number of bins or bin edges for the axis.
     range : None or tuple, optional
-        The range of the axis. If None, it will be determined based on the data.
+        The range of the axis. If None or ["min", "max"], it will return the interval [min(data), max(data)+bin_width] if bins > 1, otherwise [min(data), max(data)]. The additional bin width is added to the max value to avoid data points falling on the upper edge of the last bin, which is exclusive for boost-histogram.
```

Comment:
* If the user wants a histogram with 1 bins, it is likely to count the total number of entries, so we should not use [min(data), max(data)] which will miss the entries as max(data). What about [min(data), max(data)+abs(max(data)-min(data))] if bins=1? It gives an intuitive range of [0, 2) in case of a binary variable taking either 0 or 1 values.
* falling on -> located
* for boost-histogram -> in boost_histogram
* The case is so specific that I move it at the end of the doc line, starting a new sentence with something like "In the special case bins=1, ..."